src/promptflow/promptflow/_cli/data/chat_flow/README.md

```diff
@@ -8,7 +8,11 @@ Currently, there are two connection types supported by LLM tool: "AzureOpenAI" a
 
 ```bash
 # Override keys with --set to avoid yaml file changes
+# Create open ai connection
 pf connection create --file openai.yaml --set api_key=<your_api_key> organization=<your_organization> --name open_ai_connection
```

Comment:
remove the orgnazation part as most people won't need that

Reply:
remove it